src/components/ENSListPage/ENSListPage.css

```diff
@@ -2,6 +2,20 @@
   cursor: default;
 }
 
+.ENSListPage .empty-names {
+  display: flex;
+  width: 100%;
+  height: 200px;
+  border-radius: 12px;
+  border: 1px solid var(--divider);
+  align-items: center;
+  justify-content: center;
+  color: var(--secondary-text);
+  user-select: none;
+  text-align: center;
+  line-height: 28px;
+  font-size: 17px;
+}
```

Comment:
Should we need to add a space here?

Reply:
It is needed to have the same text format that Scene List

Reply:
Sorry I misunderstood your comment. You mean an (newline char) space between css classes not space between lines of the paragraph.  Fixed!